bokehjs/src/coffee/models/index.ts

```diff
@@ -17,3 +17,4 @@ export * from "./sources"
 export * from "./tickers"
 export * from "./tiles"
 export * from "./tools"
+export * from "./widgets"
```

Comment:
This needs to be removed. Widgets are part of a separate bundle and can't be included here.